### ISSUE TITLE
Sigmoid activation

### ISSUE THREAD

**Issue by leonormpires** [albermax/innvestigate#318]
Hi, I want to apply LRP, Grad-CAM and Guided Backpropagation to my CNN, although since I am doing a binary classification problem, I am using sigmoid activation function as the last layer. Is it not possible to apply any iNNvestigate function to my model? Will it only work with softmax?

Thanks in advance! I really need help, since this step is crucial for the conclusion of my master's thesis.

**Comment by adrhill** [comment]
Hi @leonormpires, you could try the function `model_wo_output_activation(model)` that was added in #310. 
This will require using the latest iNNvestigate version `2.1.0`.
